test/functional/wallet_bumpfee.py

```diff
@@ -60,7 +60,7 @@ def run_test(self):
 
         self.log.info("Running tests")
         dest_address = peer_node.getnewaddress()
-        test_simple_bumpfee_succeeds(rbf_node, peer_node, dest_address)
```

Comment:
commit fae14a07b6a31211ea3bd0c707754b4b743d03d2

Could mention in the commit message that this is required for the upcoming scripted diff.